Toolset/palettes/script editor/behaviors/revseeditorbehavior.livecodescript

```diff
@@ -1097,6 +1104,22 @@ on mouseDown pButtonNumber
       pass mouseDown
    end if
    
+   if sePrefGet("Editor,folding") then
+      if the commandKey is down then
+         local tLineText
+         local tAllowedLineStarts
+         put "on,private,function,command,setProp,getProp,after,before,if,repeat,switch,case,try,else" into tAllowedLineStarts
```

Comment:
Could make tAllowedLineStarts a constant and save the time of initializing a variable each time.

Reply:
thanks Mark, I guess there is more to come...

Reply:
this is a great PR though. Thanks for doing this.

Reply:
made it a constant